datadog_test.go

```diff
@@ -0,0 +1,69 @@
+package gzap
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zaptest/observer"
+
+)
+
+func expect(t *testing.T, a interface{}, b interface{}) {
+	if a != b {
+		t.Errorf("Expected %v - Got %v ", b, a)
+	}
+}
+
+func Test_Datadog(t *testing.T) {
```

Comment:
This should be Pascal Case with no `_`

Reply:
Weird, I saw other repos use the `_`.

Reply:
That might be the case, go vet didn't flag it or anything, but for consistency I don't think any of the tests in gzap do